market-service/src/main/java/rs/edu/raf/banka1/model/Listing.java

```diff
@@ -16,7 +15,7 @@
 @Getter
 @Setter
 @NoArgsConstructor
-public class ListingModel {
+public class Listing {
     @Id
```

Comment:
zasto ne autogenerated id?

Reply:
i dalje isto pitanje

Reply:
Ovo je **legacy** sad, više nam ne treba pošto sad koristimo **ListingBase**, pa sam ga ostavio samo za ovaj sprint pošto ljudi mogu da iskoriste funkcije neke od ranije za njihovu trenutnu implementaciju, pa na kraju sprinta, odnosno sutra možemo da je obrišemo.